catboost/python-package/catboost/_catboost.pyx

```diff
@@ -2522,14 +2524,17 @@ cdef class _CatBoost:
         if pool:
             dataProviderPtr = pool.__pool
         cdef TString type_name_str = to_arcadia_string(type_name)
+        cdef TString shap_mode_str = to_arcadia_string(shap_mode)
```

Comment:
Let's convert `type_name_str` and `shap_mode_str` string to enam like [here](https://github.com/catboost/catboost/blob/master/catboost/python-package/catboost/_catboost.pyx#L1092-L1094) to have only enums in c++ code.